loader/validate.go

```diff
@@ -29,6 +29,7 @@ import (
 
 // checkConsistency validate a compose model is consistent
 func checkConsistency(project *types.Project) error {
+	containerNames := map[string]string{}
```

Comment:
could use `utils.Set`, which offers `Has` method

Reply:
I can't keep the original service name with a Set only the `container_name` value